### PR TITLE
Pxc 8.0.16 packaging

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: build-ps/rpm/mysqld.cnf [hunk 1]
```diff
@@ -9,9 +9,49 @@ datadir=/var/lib/mysql
 socket=/var/lib/mysql/mysql.sock
```
Comment (EvgeniyPatlan) [inline build-ps/rpm/mysqld.cnf:9]: @vadimtk as I remember all changes to my.cnf should be approved by you. so please take a look at these changes.
We are going to change the way how we provide my.cnf and follow PS way

File: build-ps/rpm/mysqld.cnf [hunk 2]
```diff
@@ -9,9 +9,49 @@ datadir=/var/lib/mysql
...
+# Cluster connection URL contains IPs of nodes
+#If no IP is found, this implies that a new cluster needs to be created,
+#in order to do that you need to bootstrap this node
+wsrep_cluster_address=gcomm://
+
+# In order for Galera to work correctly binlog format should be ROW
+binlog_format=ROW
+
+# Slave thread to use
+wsrep_slave_threads= 8
```
Comment (vadimtk) [inline build-ps/rpm/mysqld.cnf:31]: This might be too much for some systems (wsrep_slave_threads). I would stay with default.

Comment (EvgeniyPatlan) [reply build-ps/rpm/mysqld.cnf:31]: @vadimtk currently in 5.7 we provide wsrep.cnf file with `wsrep_slave_threads= 8` (https://github.com/percona/percona-xtradb-cluster/blob/5.7/build-ps/debian/extra/wsrep.cnf#L17)
The default value according to https://galeracluster.com/library/documentation/mysql-wsrep-options.html is 1.
So should I change it to 1?

Comment (vadimtk) [reply build-ps/rpm/mysqld.cnf:31]: ok, if this is 8 in 5.7, we can use 8

**Comment by kbauskar** [comment]
I am not sure I can understand the packaging changes to the level to comment on it. So I presume it is approved from my side.

**Review by hors — APPROVED** [review]

File: build-ps/rpm/mysqld.cnf (hunk 2 above)
Comment (EvgeniyPatlan) [reply build-ps/rpm/mysqld.cnf:31]: @vadimtk are you ok with all other changes? So could I merge?

**Comment by vadimtk** [comment]
Ok to merge

**Pull request merged**